tox.ini

```
@@ -52,19 +64,23 @@ passenv =
     #     export PYENCHANT_LIBRARY_PATH=/opt/homebrew/lib/libenchant-2.2.dylib
     PYENCHANT_LIBRARY_PATH
 commands =
-    !lint-!all : python -m sphinx {[docs]sphinx_args} -b html . {[docs]build_dir}/html
+    !lint-!all-!translate : python -m sphinx {[docs]sphinx_args} -b html . {[docs]build_dir}/html
     lint : python -m sphinx {[docs]sphinx_args_extra} -b linkcheck . {[docs]build_dir}/links
     lint : python -m sphinx {[docs]sphinx_args_extra} -b spelling . {[docs]build_dir}/spell
-    all  : python -m sphinx {[docs]sphinx_args_extra} -b html . {[docs]build_dir}/html
+    translate : python -m sphinx -b gettext . _build/gettext
+    translate : sphinx-intl update -p _build/gettext -w 0 -l de,es,fr,zh_CN
```

The reference to the `_build` directory should definitely be updated....although I am a little less sure about the extra sphinx args. As best I can tell, we probably want to use them....or at least most of them. A major difference is this causes all files to be translated, whether they "need" it or not. I'm not familiar with this workflow but your PR description sounds like this `tox -e docs-translate` should just happen in CI....so, including `-a` shouldn't really matter.

```suggestion
    translate : python -m sphinx {[docs]sphinx_args_extra} -b gettext . {[docs]build_dir}/gettext
    translate : sphinx-intl update -p {[docs]build_dir}/gettext -w 0 -l de,es,fr,zh_CN
```